documentation/projects/proposals/dark_mode/20240313-project_proposal_dark_mode.md

```diff
@@ -0,0 +1,81 @@
+# 2024-03-13 Project Proposal
+
+**Author**: @fcoveram
+
+## Reviewers
+
+<!-- Choose two people at your discretion who make sense to review this based on their existing expertise. Check in to make sure folks aren't currently reviewing more than one other proposal or RFC. -->
+
+- [ ] @obulat
+- [ ] @zackkrida
+
+## Project summary
+
+<!-- A brief one or two sentence summary of the project's features -->
+
+Include a feature that changes the UI from light to a dark color scheme.
+
+### Motivation
+
+Dark mode is a customization level that is in the spirit of adapting the UI to
+people’s contexts. The benefits span a11y, personal preferences set on the
+devices, energy efficiency in devices, and an alternative browsing experience of
+visual content due to the background contrast.
+
+## Goals
+
+<!-- Which yearly goal does this project advance? -->
+
+Refine Search Experience
+
+## Requirements
+
+<!-- Detailed descriptions of the features required for the project. Include user stories if you feel they'd be helpful, but focus on describing a specification for how the feature would work with an eye towards edge cases. -->
+
+1. Design documentation of pages and UI components in the Design Library and any
+   other additional Figma files.
+2. Design documentation of transition from current to new frontend
+   implementation.
+
+## Success
+
+<!-- How do we measure the success of the project? How do we know our ideas worked? -->
+
+- Add the dark theme to the site by allowing visitors to change the UI through
+  three options: Light theme, Dark theme, or follow device setting.
+- Document the dark variant of every UI element in both the Design Library and
+  Storybook.
```

Comment:
```suggestion

- Add the dark theme to the site by allowing visitors to change the UI through
  three options: Light theme, Dark theme, or follow device setting.
- Document the dark variant of every UI element in both the Design Library and
  Storybook.
```

I think these are more "requirements" of the project, rather than ways of measuring success. How would we confirm that adding this functionality was beneficial? Perhaps an analytics event on the theme switcher, user feedback, etc?

Reply:
You are right. Those are requirements.

Regarding success, I don't think that user feedback is necessary for this project. Switching the UI's theme to fit users' preference from the device or site setting feels similar to building a responsive site or designing with a11y in mind. It adapts content to user's context.

It doesn't add, modify, or remove features or content, but it expands the way of how people use Openverse.

The above doesn't imply leaving the section empty, but after stating the above, I would approach the success from a technical perspective. What do you think?

Reply:
I'm not against of adding it. I will include the action in the section.